pkg/rc/auditing_transaction.go

```diff
@@ -0,0 +1,95 @@
+package rc
+
+import (
+	"context"
+
+	"github.com/square/p2/pkg/audit"
+	pc_fields "github.com/square/p2/pkg/pc/fields"
+	"github.com/square/p2/pkg/store/consul/transaction"
+	"github.com/square/p2/pkg/types"
+	"github.com/square/p2/pkg/util"
+)
+
+type auditingTransaction struct {
+	ctx           context.Context
+	nodes         map[types.NodeName]struct{}
```

Comment:
why is this a map of `types.NodeName` to `struct{}`? I've been using `map[types.NodeName]bool` to create sets for fast lookup. is this analogous?